components/shared_code/tests/shared_data_model/meta/test_metric.py

```diff
@@ -12,9 +12,29 @@ class MetricTest(MetaModelTestCase):
 
     def test_addition(self):
         """Test that an invalid addition value throws a validation error."""
-        message = "value is not a valid enumeration member; permitted: 'max', 'min', 'sum'"
-        self.check_validation_error(message, addition="invalid")
+        self.check_validation_error("Input should be 'max','min' or 'sum'", addition="invalid")
```

Comment:
Shouldn't there be an extra space after the comma? I guess I missed it in the source change :-)

Reply:
I think this is a bug in Pydantic!